### PR TITLE
[kai/intake-service-create] Add intake service creation

### PR DESCRIPTION
**Pull request by kai-wei-mo**
## Notion ticket link

[Create new intake](https://www.notion.so/uwblueprintexecs/Create-new-intake-3228fac4fb224ed5a978f69c13c25763)

## Implementation description
* Implemented by following the ticket description

## Steps to test
1. Visually determine correctness of changes
2. ci/pytest are currently passing

## What should reviewers focus on?
* Everything has equal priority

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by kai-wei-mo** [comment]
Wait for #97 to be merged before reviewing this PR @jennchenn

File: backend/python/app/services/implementations/intake_service.py [hunk 1]
```diff
@@ -1,10 +1,53 @@
+from ...models import db
```
Comment (jennchenn) [inline backend/python/app/services/implementations/intake_service.py:1]: i think `interfaces/intake_service.py` needs to be updated a bit as well!

**Review by jennchenn — COMMENTED** [review]
going to take a deeper look once the other PR is merged but looks good so far!

File: backend/python/app/services/implementations/intake_service.py (hunk 1 above)
Comment (kai-wei-mo) [reply backend/python/app/services/implementations/intake_service.py:1]: just to clarify: which parts of `interfaces/intake_service.py` need attention?

Comment (jennchenn) [reply backend/python/app/services/implementations/intake_service.py:1]: oops didn't see your changes to it earlier, it looks good now

**Review by jennchenn — APPROVED** [review]
nice work!

**Pull request merged**